### ISSUE TITLE
release channel

### ISSUE THREAD

**Issue by travi** [FormidableLabs/spectacle#775]
i'm looking forward to the new version once it's ready, but it looks like you've released the alpha on the latest channel. 

```
$ npm dist-tags ls spectacle
alpha: 6.0.0-alpha.2
latest: 6.0.0-alpha.3
```

until it's ready, would you mind updating the dist-tags to point to the proper versions so that its not necessary to `npm install spectacle@5` to get the stable vesion? also, `npm install spectacle@alpha` would not install the latest alpha version

**Comment by carlos-kelly** [comment]
Thank you for bringing this up. We have re-published stable and it should install 5.7.2.

**Comment by travi** [comment]
thanks. in case it's helpful in the future, it's also possible to simply [reassign the tags](https://docs.npmjs.com/cli/dist-tag) so that you don't to go through the hassle of another full publish of the same content.
